spec/factories/form526_submissions.rb

```diff
@@ -2,11 +2,13 @@
 
 FactoryBot.define do
   factory :form526_submission do
-    user_uuid { SecureRandom.uuid }
+    transient do
+      user { create(:disabilities_compensation_user) }
```

Comment:
Rather than create an entirely random `user_uuid`, use the `user`'s uuid (which is more realistic since `user` is used to populated `auth_headers_json`).